modules/javafx.base/src/main/java/module-info.java

```diff
@@ -50,7 +50,7 @@
         javafx.graphics,
         javafx.fxml,
         javafx.media,
-        javafx.swing;
+        javafx.swing, javafx.web;
```

Comment:
Minor: can you move `javafx.web` onto its own line to match the formatting in the rest of the file?